source/developer/web-service.md

```diff
@@ -1,178 +0,0 @@
-# Web Service API Overview 
```

Comment:
Is removing this page going to break links? Should we just add a tomb stone linking to api.mattermost.com and remove it from the docs sidebar, rather then entirely deleting it?